xml/System.Diagnostics/ActivityTagsCollection.xml

```diff
@@ -100,8 +111,8 @@
         <Parameter Name="item" Type="System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;" />
       </Parameters>
       <Docs>
-        <param name="item">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="item">Key and value pair of the tag to add to the collection.</param>
+        <summary>Adds an item to the collection</summary>
         <remarks>To be added.</remarks>
```

Comment:
>remarks [](start = 9, length = 7)

This method can throw InvalidOperationException if the input key existed in the listed or throw ArgumentNullException if the key is null.

this comment apply to the other Add overload too.

Reply:
```suggestion
        <remarks>To be added.</remarks>
        <exception cref="T:System.InvalidOperationException">
          <paramref name="item" /> already exists in the list.</exception>
        <exception cref="T:ArgumentNullException">
          <paramref name="item" /> is <see langword="null" />.</exception>
```